lib/room.js

```diff
@@ -271,7 +271,10 @@ class Room extends EventEmitter {
  */
 
 /**
- * A {@link RemoteTrack} was published by a {@link RemoteParticipant} to the {@link Room}.
+ * A {@link RemoteTrack} was published by a {@link RemoteParticipant} after
+ * connecting to the {@link Room}. This event is not emitted for
+ * {@link RemoteTrack}s that were published while the {@link RemoteParticipant}
+ * was connecting to the {@link Room}.
```

Comment:
If we make the change I suggest above, I recommend we include a similar example here.